[2024-03-11T14:00:00.354+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2021-07-31T00:00:00+00:00 [queued]>
[2024-03-11T14:00:00.477+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2021-07-31T00:00:00+00:00 [queued]>
[2024-03-11T14:00:00.494+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-11T14:00:01.700+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): third_task> on 2021-07-31 00:00:00+00:00
[2024-03-11T14:00:01.835+0000] {standard_task_runner.py:60} INFO - Started process 2772 to run task
[2024-03-11T14:00:02.077+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'scheduled__2021-07-31T00:00:00+00:00', '--job-id', '695', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp3keuvrrq']
[2024-03-11T14:00:02.165+0000] {standard_task_runner.py:88} INFO - Job 695: Subtask third_task
[2024-03-11T14:00:02.979+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.third_task scheduled__2021-07-31T00:00:00+00:00 [running]> on host 26e63b1d0d39
[2024-03-11T14:00:03.941+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2021-07-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-07-31T00:00:00+00:00'
[2024-03-11T14:00:03.948+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-11T14:00:03.955+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and I ran after task1 and at the same time as task 2']
[2024-03-11T14:00:04.248+0000] {subprocess.py:86} INFO - Output:
[2024-03-11T14:00:04.288+0000] {subprocess.py:93} INFO - hey, I am task3 and I ran after task1 and at the same time as task 2
[2024-03-11T14:00:04.310+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-11T14:00:04.612+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, execution_date=20210731T000000, start_date=20240311T140000, end_date=20240311T140004
[2024-03-11T14:00:05.260+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T14:00:05.588+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
